src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotAddSchemaByURLTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Test.Utilities;
+using Xunit;
+
+namespace Microsoft.NetCore.Analyzers.Security.UnitTests
+{
+    public class DoNotAddSchemaByURLTests : DiagnosticAnalyzerTestBase
```

Comment:
A "no diagnostic" unit test that adds to a Dictionary<string, string> would be good.